cmd/catalog.go

```diff
@@ -172,7 +175,7 @@ services from service catalog.
 		default:
 			fmt.Println("The following services were found:")
 			for _, component := range components {
-				fmt.Printf("- %v\n", component)
+				fmt.Printf("%v\t(plans: %v)\n", component.Name, strings.Join(component.PlanList, ","))
```

Comment:
I agree we have only 2 columns but I feel as component.Name length varies, the alignment may not look good.
So, can we move to tabwriter here?